src/modules/ens/sagas.ts

```diff
@@ -76,11 +76,9 @@ function* handleSetAlias(action: SetAliasRequestAction) {
   try {
     const client = new CatalystClient(PEER_URL, 'builder')
     const entities: Entity[] = yield client.fetchEntitiesByPointers(EntityType.PROFILE, [address.toLowerCase()])
-    const entity = entities.length > 0 ? entities[0] : null
+    const defaultEntity: Entity = yield getDefaultProfileEntity()
+    const entity = entities.length > 0 ? entities[0] : defaultEntity
```

Comment:
I think that we can avoid the default entity's request when the user already has a profile. 

```typescript
let entity: Entity
if (entities.length > 0) {
    entity = entities[0]
} else {
   entity = yield call(() => getDefaultProfileEntity())
}
```

Reply:
It looks better in that way. Fixing! Thanks